pysrc/juliacall/__init__.py

```diff
@@ -248,8 +248,37 @@ def jlstr(x):
                 "PYTHON_JULIACALL_HANDLE_SIGNALS=no."
             )
 
-init()
+    # Next, automatically load the juliacall extension if we are in a Jupyter notebook
+    CONFIG['autoload_extensions'] = choice('autoload_extensions', ['yes', 'no'])[0]
```

Comment:
Can we rename the option to `autoload_ipython_extension`? I know it's a mouthful but the current name doesn't really say what sort of extension is being loaded.

Reply:
Sure, sounds good.